info.yml

```diff
@@ -27,5 +27,5 @@ requires_core_version: "v0.14.58"
  
 # the frameworks required to run this app
 frameworks:
-    - {"name": "tk-framework-shotgunutils", "version": "v4.x.x"}
+    - {"name": "tk-framework-shotgunutils", "version": "v5.x.x"}
```

Comment:
So does this mean we are fully v5. in the dependency chain for sg-frameworks now? We had a separate ticket to fix this - can you close this out when this is merged in? 🍬 !